src/Proxy.NET/Models/RequestContext.cs

```diff
@@ -0,0 +1,28 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Proxy.NET.Models;
+
+public partial class RequestContext
```

Comment:
Should this be in the Database namespace/folder? Looks to be used in the DbContext.

Reply:
Punting to [#307](https://github.com/microsoft/azure-openai-service-proxy/issues/307) and #308

Reply:
Yes this should be in the Database folder esp as we'll planning to share the Database models between admin and proxy